src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -27,137 +30,183 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
+    >
+      <div
+        css={`
+          display: flex;
+          justify-content: space-between;
+          margin-bottom: ${6 * GU}px;
+          overflow: hidden;
+          flex-direction: ${compact ? 'column' : 'row'};
+          align-items: ${compact ? 'flex-start' : 'flex-end'};
+        `}
+      >
+        <div
+          css={`
+            display: flex;
+            align-items: center;
+          `}
+        >
           <div
             css={`
-              padding: ${6 * GU}px ${4 * GU}px ${8 * GU}px;
+              margin: 0 ${3 * GU}px 0 0;
             `}
           >
-            <div
+            <AppIcon app={{ baseUrl, icons }} size={80} />
+          </div>
+          <div>
+            <h1
               css={`
-                display: flex;
-                justify-content: space-between;
-                margin-bottom: ${6 * GU}px;
-                overflow: hidden;
+                white-space: nowrap;
+                margin-bottom: ${1 * GU}px;
+                font-size: 22px;
+              `}
+            >
+              {name}
+            </h1>
 
-                flex-direction: ${compact ? 'column' : 'row'};
-                align-items: ${compact ? 'flex-start' : 'flex-end'};
+            {author && (
+              <React.Fragment>
+                <Heading2>Created by</Heading2>
+                <div>
+                  <LocalIdentityBadge entity={author} />
+                </div>
+              </React.Fragment>
+            )}
+          </div>
+        </div>
+        <div
+          css={`
+            padding: ${compact ? `${3 * GU}px 0 0 ${80 + 3 * GU}px` : '0'};
+          `}
+        >
+          {canUpgrade && onRequestUpgrade && (
+            <Button mode="strong" onClick={onRequestUpgrade}>
+              Upgrade
+            </Button>
+          )}
+        </div>
+      </div>
+      {screenshots.length > 0 && (
+        <Screenshots repo={repo} screenshots={screenshots} />
+      )}
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          justify-content: space-between;
+          width: 100%;
+          flex-direction: ${compact ? 'column' : 'row'};
+        `}
+      >
+        <DetailsGroup compact={compact}>
+          <Heading2>Description</Heading2>
+          <div>{description}</div>
+
+          {!!repoDetails && (
+            <React.Fragment>
+              <Heading2>Details</Heading2>
+              <Markdown>
+                {
+                  /* eslint-disable react/prop-types */
+                  remark()
+                    .use(remark2react, {
+                      remarkReactComponents: {
+                        a: ({ children, ...props }) => (
+                          <SafeLink target="_blank" {...props}>
+                            {children}
+                          </SafeLink>
+                        ),
+                      },
+                    })
+                    .processSync(repoDetails).contents
```

Comment:
Should we move this in a dedicated component (e.g. `Markdown`) and cache it with `useMemo()`?